### PR TITLE
Update translation in french

### PR DESCRIPTION
**Pull request by forresst**
1. Fix experimental-scope-hoisting document (parcel-bundler/website@6701477f7054053d3de191c882f0c50c5e0f62ab)
2. Add filename strategy doc (parcel-bundler/website@5e61824fac5dc371cc1172a976f1e06818041f9e)
3. Update babel part in transforms (parcel-bundler/website@5709c099c1f78e7c82a4397593c98ce0f3332724)
4. Update event docs (parcel-bundler/website@50bd192e70e9f29334410f9e2bf65dc535128fa9)

### CODE REVIEW

File: src/i18n/fr/docs/production.md [hunk 1]
```diff
@@ -6,6 +6,26 @@ Quand vient le temps d'empaqueter votre application pour la production, vous pou
...
+
+## Stratégie de nommage des fichiers
+
+Pour permettre à votre cdn de définir des règles de mise en cache très agressives, afin d'avoir une performance et une efficacité optimale, Parcel hache les noms de fichier de la plupart des bundles (selon si le bundle doit avoir un nom lisible/mémorisable ou non, principalement pour le référencement).
+
+Parcel suit le tableau suivant, lorsqu’il s’agit de nommer des bundles. (Les points d'entrée ne sont jamais hachés)
+
+| Type de bundle | Type | Contenu haché |
+| ---:| --- |:---:|:---:|
+| Any | point d'entrée          | ❌ |
```
Comment (fathyb) [inline src/i18n/fr/docs/production.md]: Any -> Tous (ou Tout je me rappel plus, je te laisse choisir haha)

File: src/i18n/fr/docs/production.md [hunk 2]
```diff
@@ -6,6 +6,26 @@ Quand vient le temps d'empaqueter votre application pour la production, vous pou
...
+| JavaScript | `<script>`       | ✅ |
+| JavaScript | Import dynamique | ❌ |
+| JavaScript | Service worker   | ❌ |
+| HTML | iframe                 | ❌ |
+| HTML | lien d'ancrage         | ❌ |
+| Raw (Images, fichiers texte, ...) | Import/Require/... | ✅ |
```
Comment (fathyb) [inline src/i18n/fr/docs/production.md]: Raw -> Brut

File: src/i18n/fr/docs/transforms.md [hunk 3]
```diff
@@ -11,17 +11,29 @@ Cela fonctionne même dans des `node_modules` tiers : si un fichier de configura
...
+    "react"
+  ]
 }
 ```
 
+### Transformations babel par défaut
+
+Parcel transpile par défaut votre code avec `babel-preset-env`, ceci transpile chaque module interne (requires locaux) et externe (node_modules) pour correspondre à la cible définie.
+
+Pour la cible `browser`, il utilise [browserlist](https://github.com/browserslist/browserslist), la browserlist cible peut être définie dans le fichier `package.json`(`engines.browsers` ou `browserslist`) ou en utilisant un fichier de configuration (`browserslist` ou `.browserslistrc`).
```
Comment (fathyb) [inline src/i18n/fr/docs/transforms.md]: Il manque un espace entre `package.json` et `('engines.browsers`

**Comment by forresst** [comment]
@fathyb  Thank you for your review. It's fixed

**Pull request merged**